### DISCUSSION TITLE
npm run build fails with "SassError: Undefined variable"

### DISCUSSION THREAD

**Discussion by Nivedita-coder** [twbs/bootstrap#38644] [Q&A]
npm run build fails with the following error:

`SassError: Undefined variable.`
`110 │   background: $background-color;`
`    │               ^^^^^^^^^^^^^^^^^`
`src/assets/css/sass/_cern.style.scss 110:15             @import`
`src/assets/css/sass/main.scss 2:9                       @import`
` src/assets/css/sass/themes/cern.dark.bluenavy.scss 1:9  root stylesheet`

I've tried to do with `@import "~bootstrap/scss/utilities"` but it didn't worked. 

Any idea what am I doing wrong?

**Comment by julien-deramond** [comment]
Hey @Nivedita-coder 
`$background-color` Sass var doesn't come from Bootstrap so it might come from your own Scss files, or from one of the theme that you're using.
Do you have other elements to share?

**Reply by Nivedita-coder** [reply]
Hey @julien-deramond,
Thanks for your reply!

Actually, I don't have other elements to share related to this issue. Even I've tried importing themes folder to specific file and many other ways but so far no luck.

**Reply by julien-deramond** [reply]
Unfortunately, without further elements from your own code or the theme used, I won't be able to help more than saying that `$background-color` doesn't exist in Bootstrap so the problem is probably elsewhere.

In order to just fix the Sass compilation, you can just try to define a random value (e.g. yellow) for this var and check what's the impact of this modification 🤷 

```scss
$background-color: #fd0;
```

or maybe try to search `$background-color` text in your `node_modules` directory to see if it comes from somewhere.

**Comment by countaustin1990** [comment]
Restart the project all over using the instructions in bootstrap websites. Follow each instructions. Don't try to doing it from your head you won't get it.
